recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set commit = "b409536eb9029b274d2956ec14a7954a0089aa3a" %}
-{% set version = "0.3.0" %}
+{% set version = "0.2.0.213.g296fce14" %}
```

Comment:
This is wrong. Do `git fetch upstream_remote_name --tags` to fetch the latest tags